lib/uiautomator2.js

```diff
@@ -102,40 +104,18 @@ class UiAutomator2Server {
   }
 
   async getPackageName (apk) {
```

Comment:
Naming here seems to be mixed up. `getPackageName` and `getAPKVersion` take an apk, while `getInstalledPackageVersion` takes a package. Can these be normalized? Perhaps `getAPKName`? The call in `installServerApk` seems to be the only place it is used.